suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -35,27 +39,29 @@ const TodayHeaderCell = ({
     network,
     historicalRates,
 }: TodayHeaderCellProps) => {
-    const coins = useSelector(selectCoinsLegacy);
-    const fiatCurrency = useSelector(selectFiatCurrency);
-    const currentRates = coins.find(coin => coin.symbol === network)?.current?.rates;
+    const fiatCurrencyCode = useSelector(selectFiatCurrencyCode);
+    const fiatRateKey = getFiatRateKey(network, fiatCurrencyCode);
+    const currentRates = useSelector((state: FiatRatesRootState) =>
+        selectFiatRatesByFiatRateKey(state, fiatRateKey),
+    );
```

Comment:
This could be a new selector which gets the fiat rate key itself. It's in couple of places in this PR